pkgs/top-level/all-packages.nix

```diff
@@ -35928,6 +35928,8 @@ with pkgs;
 
   bcompare = libsForQt5.callPackage ../applications/version-management/bcompare {};
 
+  xp-pen-deco-01-v2-driver = libsForQt5.callPackage ../misc/drivers/xp-pen-deco-01-v2-driver {};
```

Comment:
Versioning should also not be apart of the top-level attribute name, and the path in nixpkgs.

Also, I think this should be moved to `pkgs/os-specific/linux`.